CompetitionTemplate/src/competition/subsystems/turret/rotation/commands/RotateTurretWithGamepadCommand.java

```diff
@@ -25,6 +29,13 @@ public void initialize() {
 
     @Override
     public void execute() {
-        rotationSubsystem.setRotationPower(oi.leftJoystick.getButton(8) ? oi.leftJoystick.getRawAxis(4) : 0);
+        double leftIntent = leftTrigger.getVector().y;
+        double rightIntent = -1 * rightTrigger.getVector().y;
+        
+        if(rightIntent != 0 && leftIntent != 0){
```

Comment:
I'd be worried about the case where you are trying to fine-tune aim. I think it would be easily possible for you to (either intentionally or unintentionally) press both of them partially, and as soon as you do that the turret would just stop moving entirely. In my mind, having both triggers partially depressed is a completely valid state.